internal/cli/delete.go

```diff
@@ -10,10 +10,19 @@ var ()
 
 // CmdAppDelete implements the command: epinio app delete
 var CmdAppDelete = &cobra.Command{
-	Use:               "delete NAME1 [NAME2 ...]",
-	Short:             "Deletes one or more applications",
-	Args:              cobra.MinimumNArgs(1),
-	ValidArgsFunction: matchingAppsFinder,
+	Use:   "delete NAME1 [NAME2 ...]",
+	Short: "Deletes one or more applications",
+	Args:  cobra.MinimumNArgs(1),
+	ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
How is `matchingAppsFinder` different? Also, I see the methods are missing the `DisableVersionWarning()`. A rebase needed?

Reply:
`matchingAppsFinder` completes the first argument of the command as app name. The method here completes any argument as app name - It is missing the restriction to the first argument, the check for `len(args)`.

Yes, rebase is needed. Will do.

Reply:
Done (Rebase, and fix): Commit 1c9a378f99f35c944a48e05991397cc558fb0f8b